openedx/core/djangoapps/oauth_dispatch/views.py

```diff
@@ -134,6 +134,25 @@ class AccessTokenExchangeView(_DispatchingView):
     """
     dot_view = auth_exchange_views.DOTAccessTokenExchangeView
 
+    def dispatch(self, request, *args, **kwargs):
+        response = super().dispatch(request, *args, **kwargs)
```

Comment:
@jawad-khan: Reminder to confirm through a unit test that this won't work for a user with a disabled account. Thanks.

Reply:
@jawad-khan: I think this is the only blocker. Otherwise, looks good. Thanks!

Reply:
Is it possible to add a unit test to confirm and ensure this can't regress?

Reply:
done.